.babelrc

```diff
@@ -1,8 +1,3 @@
 {
-  "presets": ["react", "env", "stage-1"],
-  "env": {
-    "development": {
-      "presets": ["react-hmre"]
-    }
-  }
+  "presets": ["react", "es2015", "stage-0"]
```

Comment:
Would you mind clarifying the reasoning behind these changes?

As far as I understand we should aim to use `react` and `env` only. Another point is that `react-hmre` it's unfamiliar to me and I fear something breaks up removing it. I think is related to the hot module reload feature but I'm not sure. Sorry for my uncertainties but I only modified this project once a long time ago.

Reply:
Why should we avoid other babel presets?

Reply:
It's not of a big deal. But to answer your question: to avoid having more ways to code the same things. The more versions of JS flavors we support, the more inconsistency in codebase we might have. Some devs will use one approach, some others. I prefer to stick with just a few styles and rules so anyone can quickly recognize any part of the code.

Anyways, we don't have any specific agreement on JS flavors so it can be revisited later.